crates/ui/src/components/animation_frame_view.rs

```diff
@@ -236,13 +236,14 @@ impl AnimationFrameView {
             })
```

Comment:
Opening the animation editor shows this error:
![image](https://github.com/user-attachments/assets/3986fa66-278f-4e0e-b223-806fd4329a19)